docs/sources/auth/_index.md

```diff
@@ -1 +1,14 @@
+---
+title: Authentication setup
+menuTitle: Authentication setup
+description: Documentation reference for identity providers and authentication services.
+aliases:
```

Comment:
Has this file moved? If not, then we don't need an alias at the moment. Said another way, aliases should only document where the file used to exist and do not need to document its current location.

Reply:
Changed.